components/story-element.js

```diff
@@ -71,6 +71,12 @@ function StoryElementTable({element}) {
   return React.createElement(Table, { id: element.id, data: element.data, hasHeader: element.metadata['has-header']});
 }
 
+function StoryElementFile({element}) {
+  return React.createElement("div", {className: "attachment", key: element.id}, 
```

Comment:
is not `React.createElement` redundant? `<div />` should do right?  correct me if I am wrong, I am also noob in react